react/src/components/atoms/buttons/ButtonAlert/index.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+import Icon from '../../icons/Icon';
+
+
+const ButtonAlert = ({
+  text, onClick, hideText, showText
+}) => (
+  <button className="ma__button-alert" onClick={onClick}>
```

Comment:
This is missing type:
```suggestion
  <button className="ma__button-alert" onClick={onClick} type="button">
```